src/backend/src/prisma/seed.ts

```diff
@@ -245,6 +248,34 @@ const performSeed: () => Promise<void> = async () => {
   /** If the .env file exists, set the FINANCE_TEAM_ID */
   if (currentEnv) {
     currentEnv.FINANCE_TEAM_ID = financeTeam.teamId;
+
+    /** Updating slack ids of Thomas Emrax and Regina George */
+    prisma.user.update({
```

Comment:
if this doesnt exist its gonna fail to seed i think right @RChandler234 maybe it should be currentEnv.SLACK_ID ?? ‘’

Reply:
Yeah if it's undefined we should just return the old slack id we were using, also is there a reason we can't just do this where the data for the user is being declared instead of updating it later like this?